File: java/delivery-api/GetTaxonomyGroups.java
// DocSection: delivery_api_get_taxonomy_groups
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

// Gets 3 taxonomy groups
CompletionStage<List<TaxonomyGroup>> taxonomies = client.getTaxonomyGroups(
    DeliveryParameterBuilder.params()
        .page(null, 3)
        .build()
    ).thenApply(taxonomyGroupListingResponse -> taxonomyGroupListingResponse.getTaxonomies());
// EndDocSection

File: php/delivery-api/GetTaxonomyGroups.php
<?php
// DocSection: delivery_api_get_taxonomy_groups
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('<YOUR_ENVIRONMENT_ID>');

$taxonomyGroups = $client->getTaxonomies((new QueryParams())
            ->limit(3));
// EndDocSection
?>